tools/scripts/fuzz.sh

```diff
@@ -43,7 +43,7 @@ mkdir -p fuzz_crashers/original
 export ASAN_OPTIONS="dedup_token_length=10"
 
 echo "running"
-nice "./bazel-bin/test/fuzz/$what" \
+"./bazel-bin/test/fuzz/$target" \
```

Comment:
You want to `nice` the fuzzer. That deprioritizes the fuzzer process, so if other things need CPU time they will get it over the fuzzer. Otherwise, it'll take resources from other things you're doing on your devbox (e.g., running Sorbet with VS Code).

Reply:
ack. should I add nice to the other script, then? I rm'd it here because it wasn't there.

Reply:
Nah, because the other script is trying to minimize the input over a *wallclock time* budget (e.g. 10 minutes), so if you steal CPU from it it'll do a strictly worse job.